ocs_ci/deployment/deployment.py

```diff
@@ -278,6 +278,12 @@ def deploy_ocs_via_operator(self):
             deviceset_data['dataPVCTemplate']['spec'][
                 'storageClassName'
             ] = constants.DEFAULT_SC_VSPHERE
+            disk_size = int(
+                config.ENV_DATA.get('disk_size', constants.DEFAULT_DISK_SIZE)
+            )
+            deviceset_data['dataPVCTemplate']['spec']['resources']['requests'][
```

Comment:
cannot we just rely on device_size for all the platforms? The setup is already done in lines:
https://github.com/red-hat-storage/ocs-ci/pull/937/files#diff-153ccf5916ca230718b5455c349afbcdR256-R261
The old style of deployment will be deleted so we will have only operator deployment anyway.

Reply:
changed the variable name in all places